package.json

```diff
@@ -31,7 +31,7 @@
     "standard": "^17.1.2",
     "stylelint": "^16.11.0",
     "stylelint-config-gds": "^2.0.0",
-    "vite": "^5.4.11",
+    "vite": "^6.0.3",
```

Comment:
This change makes the asset loading fail at run time (specifically images and fonts from the govuk-frontend package won't load). I'm still trying to figure out:
- why that's broken
- whether we can get the build to fail when this happens

Reply:
We get these relevant warnings in the frontend build task:

```
$govuk-fonts-path + $filename referenced in $govuk-fonts-path + $filename didn't resolve at build time, it will remain unchanged to be resolved at runtime

$govuk-images-path + $filename referenced in $govuk-images-path + $filename didn't resolve at build time, it will remain unchanged to be resolved at runtime
```

Reply:
Narrowed it down slightly - Vite 6 uses the 'modern' Sass API by default, previous versions used the 'legacy' API. Switching back to legacy solves the issue, but the legacy API is deprecated so ideally I'd like to figure out why the modern one breaks our assets

Reply:
Great catch, thank you for finding this and looking into it. Out of interesting, how did you realise it was broken?

Reply:
@stephencdaly spotted that the release notes on [the latest vitest bump](https://github.com/alphagov/forms-admin/pull/1643) contained this line:
> Revert support for Vite 6

I was testing locally to see if anything weird happened when combining the two and noticed this warning in the logs